Corrected email addresses, spacing, added SAP

RoAnnC1 opened the pull request:
Charlie please review.

Commit:
Merge pull request #1 from concur/preview

Merge from preview

Commit:
Updated email addresses, added SAP

Made a few additional changes to fix spacing issues. Page needs some additional rewrites.

Commit:
Corrected title

charlieconcur reviewed (approved):
Looks good to me...!

Commit:
Merge pull request #2 from concur/preview

Update fork

Commit:
Updated email, added SAP, other minor edits.

sarraloew reviewed (approved):
Thank you!

Commit:
Update e-receipt, add SAP, minor edits.

There are still a couple of instances of Concur that I didn't change to SAP Concur that refer to the UI and I don't know if that's been updated.

Commit:
Fixed some additional issues.

charlieconcur commented on src/api-guides/e-receipts.markdown:
```diff
@@ -1,109 +1,108 @@
 ### User Connections / Authentication
 
-The below Authentication Methods are available to obtain an access token.  Password Grant or Authorization Grant or One-Time Password Grant is required for Certification:
+;The below Authentication Methods are available to obtain an access token. Password Grant or Authorization Grant or One-Time Password Grant is required for Certification:
```
Do we need the semicolon here?

RoAnnC1 replied:
No, sorry. Not sure what happened there.

charlieconcur commented on src/api-guides/e-receipts.markdown:
```diff
@@ -1,109 +1,108 @@
-- [Password Grant](/api-reference/authentication/apidoc.html#password_grant) – Connect from the Concur App Center.
-- [Authorization Grant](/api-reference/authentication/apidoc.html#auth_grant) – Connect from the Partner App.
-- [One-Time Password Grant](/api-reference/authentication/apidoc.html#otp_grant) – Connect from the Partner App utilizing Email.
+- [Password Grant](/api-reference/authentication/apidoc.html#password_grant) &ndash; Connect from the SAP Concur App Center.
```
While I agree that it probably looks a bit nicer to use the &ndash; -- in practical terms it just makes the docs harder to read and edit. I would move these back to plain dash.

RoAnnC1 replied:
Changed back to hyphen.

charlieconcur commented on src/api-guides/e-receipts.markdown:
```diff
@@ -1,109 +1,108 @@
-5. When your application receives the redirect call strip the **id** and **requestToken** values from the URI and use those on a Post request to the Concur Authorization service to obtain the official oauth2 accesstoken and refreshtoken for the user using the [password grant](/api-reference/authentication/apidoc.html#password_grant) while [being Geo Aware](#being_geo_aware).
-6. Decode the **id\_token** to obtain the **sub** value and store this value as the user **id** (see [https://jwt.io](https://jwt.io))
-7. An access token is valid only for one hour.  The access token should be cached in memory and discarded after use.
+5. When your application receives the redirect call strip the **id** and **requestToken** values from the URI and use those on a Post request to the SAP Concur Authorization service to obtain the official OAuth2 **accessToken** and **refreshToken** for the user using the [password grant](/api-reference/authentication/apidoc.html#password_grant) while [being Geo Aware](#being_geo_aware).
```
When noting key names or values use tick marks:

```
`id`
`requestToken`
`accessToken`
`refreshToken`
```

RoAnnC1 replied:
Changed

charlieconcur commented on src/api-guides/e-receipts.markdown:
```diff
@@ -1,109 +1,108 @@
-5. When your application receives the redirect call strip the **id** and **requestToken** values from the URI and use those on a Post request to the Concur Authorization service to obtain the official oauth2 accesstoken and refreshtoken for the user using the [password grant](/api-reference/authentication/apidoc.html#password_grant) while [being Geo Aware](#being_geo_aware).
-6. Decode the **id\_token** to obtain the **sub** value and store this value as the user **id** (see [https://jwt.io](https://jwt.io))
-7. An access token is valid only for one hour.  The access token should be cached in memory and discarded after use.
+5. When your application receives the redirect call strip the **id** and **requestToken** values from the URI and use those on a Post request to the SAP Concur Authorization service to obtain the official OAuth2 **accessToken** and **refreshToken** for the user using the [password grant](/api-reference/authentication/apidoc.html#password_grant) while [being Geo Aware](#being_geo_aware).
```
Anytime you see underscores, convert them to dashes:

```
Before: #being_geo_aware
After:  #being-geo-aware
```

RoAnnC1 replied:
Changed

charlieconcur commented on src/api-guides/e-receipts.markdown:
```diff
@@ -160,14 +159,14 @@ To [revoke](/api-reference/authentication/apidoc.html#revoke_token) a user&#39;s
 China = `https://cn.api.concurcdc.cn`
 
-You will need to be aware of the geolocation where the user exists in and make the call to the APIs correctly.  If there is a case where you will not / do not know the user&#39;s geolocation, then you should make the API call using the default US Base URI `https://us.api.concursolutions.com` and expect a geolocation error which will return the correct geoLocation for the user.  Sample of Error be ...
+You will need to be aware of the geolocation where the user exists in and make the call to the APIs correctly. If there is a case where you will not/do not know the user&#39;s geolocation, then you should make the API call using the default US Base URI `https://us.api.concursolutions.com` and expect a geolocation error which will return the correct geoLocation for the user. Sample of Error below:
```
This: `&#39;` was leftover from someone who didn't know the difference between a tick mark and a single quote on their keyboard. Please convert all of these to single quotes when you find them.

RoAnnC1 replied:
Fixed

Commit:
Merge pull request #3 from concur/preview

Update from preview

Commit:
Update app-certification.markdown

Commit:
Punctuation changes

Reverted en dashes to hyphens
Changed ** to `
Corrected error
Changed _ to - in anchor links

Commit:
Fixed broken href

This page needs additional edits. Waiting for some answers from the Travel team.

RoAnnC1 commented:
Fixed changed Charlie requested
